library_conventions/packages.adoc

```diff
@@ -1,8 +1,8 @@
 [#libraryconventions-packages]
 = Package Conventions
-:ipc7351-pdf: http://pcbget.ru/Files/Standarts/IPC_7351.pdf
-:ipc7351-naming-pdf: https://www.pcblibraries.com/downloads/Guidelines!Library-Expert-Land-Pattern-Naming-Convention.asp
-:ipc7351c-slides: https://ocipcdc.org/archive/What_is_New_in_IPC-7351C_03_11_2015.pdf
+:ipc7351-pdf: https://www.ipc.org/TOC/IPC-7351.pdf
+:ipc7351-naming-pdf: https://www.cskl.de/fileadmin/csk/dokumente/produkte/pcbl/ipc_standard_pcb_library_expert_Land_Pattern_Naming_Convention.pdf
```

Comment:
I'm not sure whether we should change this link. In contrast to the IPC standard itself, the Library Expert Naming Conventions are available freely (after accepting the EULA, which is basically just a disclaimer), so I'd like to avoid unauthorized and outdated uploads if we can use the official up to date version instead.

(The changes between the 2017 version and the 2019 version are small, but it's important that everybody uses the same naming guidelines. For example, the 2017 version still includes fabrication complexity levels as part of the recommended name while the 2019 version doesn't. They are not applicable in LibrePCB anyways, since something like that should be modelled as footprint variants, and not be part of the package name.)